python/hsfs/client/rondb_rest_client.py

```diff
@@ -0,0 +1,224 @@
+#
+#   Copyright 2024 Hopsworks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+import requests
+from furl import furl
+from typing import Optional, Any
+
+from hsfs import client
+from hsfs.client.exceptions import FeatureStoreException
+from hsfs.core import variable_api
+
+_rondb_client = None
+
+
+def init_rondb_rest_client(optional_config: Optional[dict[str, Any]] = None):
+    assert (
+        client.get_instance() is not None
+    ), """Hopsworks Client is not connected. Please connect to Hopsworks cluster
+            via hopsworks.login or hsfs.connection before initialising the RonDB REST Client.
+            """
+
+    global _rondb_client
+    if not _rondb_client:
+        _rondb_client = RondbRestClientSingleton(optional_config=optional_config)
+    else:
+        _rondb_client.refresh_rondb_connection(optional_config=optional_config)
```

Comment:
each time `init_rondb_rest_client` is called in `init_serving`, the connection is refreshed. It is common that multiple fv are used, but `refresh_rondb_connection` is not necessary for each `init_serving`.

Reply:
Yes I think this will be addressed with the comment above